src/osd/PGLog.cc

```diff
@@ -993,6 +993,12 @@ void PGLog::read_log(ObjectStore *store, coll_t pg_coll,
       if (i->version <= info.last_complete) break;
       if (cmp(i->soid, info.last_backfill, info.last_backfill_bitwise) > 0)
 	continue;
+      std::map<hobject_t, pg_missing_t::item, hobject_t::ComparatorWithDefault>::iterator miter = missing.missing.find(i->soid);
+      if (miter != missing.missing.end()) {
+        assert(did.count(i->soid));
+        miter->second.omap_unchanged = miter->second.omap_unchanged && i->omap_unchanged;
+        miter->second.dirty_extents.intersection_of(i->dirty_extents);
```

Comment:
currently dirty_extents indicate object interval which is not modified. Thus, we need intersection_of here. I think the name dirty_extents would be confused here. The name, like clean_extents, would be better?